sotodlib/tod_ops/demodulation.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
+from sotodlib.tod_ops import filters
+from scipy.optimize import curve_fit
+import matplotlib.pyplot as plt
+
+
+def _remove_demod_placeholders(aman):
+    '''Remove some temporary data from aman that was used for demodulation'''
+    if 'highband_sine' in aman:
+        aman.move('highband_sine', None)
+    if 'highband_cos' in aman:
+        aman.move('highband_cos', None)
+    if 'mod_sine' in aman:
+        aman.move('mod_sine', None)
+    if 'mod_cos' in aman:
+        aman.move('mod_cos', None)
+    return
+
+
+def demod_sine(aman, freq=8.0, lp_fc=1, signal_name='signal',
```

Comment:
We started using a different way of passing in arguments in sotodlib than I originally used in latrt-testing. We also standardized the detector axis name to dets. 

For the two demodulation functions could you:

1. remove the `dets_axis` keyword arg and replace `aman['dets_axis']` with just `aman.dets` and `'det_axis'` with `'dets'`.

2.  replace `signal_name='signal'` with `signal=None`, replace `aman[signal_name]` with `signal` and have 
```
if signal is None:
    signal = aman.signal
```
at the beginning of the functions.

[Example of how we've started doing this](https://github.com/simonsobs/sotodlib/blob/master/sotodlib/tod_ops/jumps.py#L350).

Reply:
Sounds good. I think I changed these to what you mentioned. Is the 'samps' axis name used here also universal?